### ISSUE TITLE
Application will not start on Ubuntu Jammy (22.04)

### ISSUE THREAD

**Issue by NetForces** [ImpulseAdventure/GUIslice-Builder#223]
Tried the released binary as well as building from source (master branch). On both I get:

```
courchea@X1-Andre:{9:06}~/Downloads/GUIslice-Builder/builder/build/tmp/linux-x64/GUIsliceBuilder:master ✗ ➭ ./GUIslice.sh 
java.nio.file.NoSuchFileException: ./logs/builder.log.lck
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newFileChannel(UnixFileSystemProvider.java:181)
	at java.base/java.nio.channels.FileChannel.open(FileChannel.java:298)
	at java.base/java.nio.channels.FileChannel.open(FileChannel.java:357)
	at java.logging/java.util.logging.FileHandler.openFiles(FileHandler.java:512)
	at java.logging/java.util.logging.FileHandler.<init>(FileHandler.java:308)
	at builder@0.17.b16/builder.controller.LogManager.openLogger(LogManager.java:67)
	at builder@0.17.b16/builder.Builder.main(Builder.java:205)
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.util.logging.FileHandler.setFormatter(java.util.logging.Formatter)" because "this.fh" is null
	at builder@0.17.b16/builder.controller.LogManager.openLogger(LogManager.java:75)
	at builder@0.17.b16/builder.Builder.main(Builder.java:205)
```

**Comment by Pconti31** [comment]
@NetForces Access ACL issue maybe?  Check permissions on GUIsliceBuilder and all sub-directories.
cd to GUIsliceBuilder/logs
do a **touch ./builder.log.lck**
If it creates it do a rm ./builder.log.lck so the java doesn't think its still open.

**Comment by NetForces** [comment]
No `logs` folder:

```
courchea@X1-Andre:{12:42}~/Downloads/GUIsliceBuilder ➭ ll
total 40K
drwxr-xr-x  2 courchea courchea 4.0K Nov 21 15:52 arduino_res
drwxr-xr-x  2 courchea courchea 4.0K Nov 21 15:52 bin
drwxr-xr-x  4 courchea courchea 4.0K Nov 21 15:52 conf
drwxr-xr-x  9 courchea courchea 4.0K Nov 21 15:52 fonts
-rwxr-xr-x  1 courchea courchea  165 Nov 21 15:52 GUIslice.sh
drwxr-xr-x 16 courchea courchea 4.0K Nov 21 15:52 legal
drwxr-xr-x  4 courchea courchea 4.0K Nov 21 15:52 lib
drwxr-xr-x  2 courchea courchea 4.0K Nov 21 15:52 linux_res
-rw-r--r--  1 courchea courchea  337 Nov 21 15:52 release
drwxr-xr-x  4 courchea courchea 4.0K Nov 21 15:52 templates
```

**Comment by NetForces** [comment]
Created the `logs` folder and it started.

**Comment by Pconti31** [comment]
@NetForces Thanks!  Turns out the problem was with my build.gradle script.
I had this for linux:
```
  from (DISP_LINUX_X64) {
   excludes = ["**/*.sh", "**/bin/java"]
   fileMode 0644
  }
	
```
When I needed:
```
  from (DISP_LINUX_X64) {
   includeEmptyDirs = true
   excludes = ["**/*.sh", "**/bin/java"]
   fileMode 0644
  }
```
I'll fix this with a new release 0.17.b17
Paul--

**Comment by Pconti31** [comment]
Fixed in Release 0.17.b17
